robottelo/ui/computeresource.py

```diff
@@ -587,12 +587,6 @@ def host_provisioning_check(self, ip_addr):
             raise ProvisioningCheckError(
                 'Failed to ping virtual machine Error:{0}'.format(
                     result.stdout))
-        ssh_check = ssh.command(
```

Comment:
is not needed anymore?

Reply:
It takes lot of time for the box to be created on rhev and test test will always timeout before the bootstrap of the client is done. We will need to verify this with using wrapapi directly quering the rhev server.